### DISCUSSION TITLE
Philips Hue Go (Bluetooth)

### DISCUSSION THREAD

**Discussion by madfx-cpu** [Koenkk/zigbee2mqtt#12223] [Devices]
Appears the model has been updated so while the old model is supported the new one cannot be added

7602031PU is the new model, I didn't realise it was different when I ordered it until it arrived.

Hoping this can be added, if more info is required please let me know :)

**Comment by Koenkk** [comment]
Please fill in https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=new+device+support&template=new_device_support.yaml&title=%5BNew+device+support%5D%3A+

**Reply by madfx-cpu** [reply]
Thanks, have submitted that info through now :)
